src/ILCompiler.Compiler/src/Compiler/Compilation.cs

```diff
@@ -257,11 +257,16 @@ public void RootStaticBasesForType(TypeDesc type, string reason)
                     }
                 }
             }
-
-            public void RootVirtualMethodUse(MethodDesc method, string reason)
+            
+            public void RootMethodForReflection(MethodDesc method, string reason)
```

Comment:
Nit: would RootVirtualMethodForReflection (with an assert the method is virtual) be a better name?